src/test/java/com/acrolinx/sidebar/pojo/settings/AcrolinxSidebarInitParameterTest.java

```diff
@@ -0,0 +1,54 @@
+/* Copyright (c) 2018-present Acrolinx GmbH */
+package com.acrolinx.sidebar.pojo.settings;
+
+import java.util.Collections;
+import java.util.Properties;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class AcrolinxSidebarInitParameterTest
+{
+    private static final String OS_NAME = "os.name";
+    private static final String OS_VERSION = "os.version";
+    private static final String JAVA_VENDOR = "java.vendor";
+    private static final String JAVA_VM_NAME = "java.vm.name";
+    private static final String JAVA_RUNTIME_NAME = "java.runtime.name";
+    private static final String JAVA_RUNTIME_VERSION = "java.runtime.version";
+
+    @Test
+    void toStringTest()
+    {
+        Properties properties = (Properties) System.getProperties().clone();
```

Comment:
maybe make this local variable final to make the idea more clear and avoid accidental re-assigning further down